independent-projects/tools/devtools-common/src/main/java/io/quarkus/devtools/commands/handlers/CreateProjectCommandHandler.java

```diff
@@ -87,14 +86,19 @@ public QuarkusCommandOutcome execute(QuarkusCommandInvocation invocation) throws
                         .messageWriter(invocation.log())
                         .build();
                 invocation.log().info("-----------");
-                invocation.log().info("selected extensions: \n"
-                        + extensionsToAdd.stream().map(e -> "- " + e.getGroupId() + ":" + e.getArtifactId() + "\n")
-                                .collect(Collectors.joining()));
-                final CodestartProject codestartProject = prepareProject(input);
-                Codestarts.generateProject(codestartProject, invocation.getQuarkusProject().getProjectDirPath());
+                if (!extensionsToAdd.isEmpty()) {
+                    invocation.log().info("selected extensions: \n"
+                            + extensionsToAdd.stream().map(e -> "- " + e.getGroupId() + ":" + e.getArtifactId() + "\n")
+                                    .collect(Collectors.joining()));
+                }
+
+                final QuarkusCodestartCatalog catalog = QuarkusCodestartCatalog
+                        .fromQuarkusPlatformDescriptor(invocation.getPlatformDescriptor());
+                final CodestartProjectDefinition projectDefinition = catalog.createProject(input);
+                projectDefinition.generate(invocation.getQuarkusProject().getProjectDirPath());
```

Comment:
Where is `extensionsToAdd` used?

Reply:
@gastaldi https://github.com/ia3andy/quarkus/blob/multi-project-codestart/independent-projects/tools/devtools-common/src/main/java/io/quarkus/devtools/commands/handlers/CreateProjectCommandHandler.java#L79